packages/doenetml/src/Viewer/renderers/graph.jsx

```diff
@@ -77,7 +77,7 @@ export default React.memo(function Graph(props) {
             showNavigation: false, // will add navigation buttons later so can style them
             // keepAspectRatio: SVs.identicalAxisScales,
             zoom: { wheel: !SVs.fixAxes },
```

Comment:
Let's add `needShift: false` for zoom, too, as that's how it works for both Desmos and Geogebra.